tests/common/utils.rs

```diff
@@ -28,31 +28,31 @@ use sp_core::{crypto::*, H256};
 pub fn event_deposited(funding_id: H256, amount: u64) {
 	assert_eq!(
 		last_event(),
-		Event::pallet_perun(pallet_perun::Event::Deposited(funding_id, amount))
+		Event::Perun(pallet_perun::Event::Deposited(funding_id, amount))
```

Comment:
Can you explain the reason behind this change? Where do you define `Perun`?

Reply:
This change came from the update `3.0.0` -> `4.0.0-dev` from frame.  
The frame configuration changed and the aliases from `mock.rs` can now be used to address the pallet.